src/pages/Setup.tsx

```diff
@@ -1,12 +1,140 @@
-import React, { useState } from 'react'
-
-type hints = string[]
+import React, { useState } from "react";
+import {
+  Box,
+  Center,
+  FormControl,
+  FormErrorMessage,
+  HStack,
+  Input,
+  Text,
+  useToast,
+} from "@chakra-ui/react";
+import BoxContainer from "../components/BoxContainer";
+import PrimaryButton from "../components/PrimaryButton";
+import AppLayout from "../layouts/AppLayout";
+import { AiOutlineArrowRight } from "react-icons/ai";
+import ListHint from "../components/ListHint";
+export type hints = string[];
```

Comment:
what about type `hints` in `src/pages/Hints.tsx`

Reply:
api ka, Tine view edit soon🥺